web/src/components/invisible-wallet-landing/landing-page.tsx

```
@@ -0,0 +1,765 @@
+                  )
+                  // identifiers after import keyword is stripped
+                  .replace(
+                    /\b(useInvisibleWallet|createWallet|wallet|platformId|defaultNetwork)\b/g,
+                    '<span style="color:#82aaff">$1</span>'
+                  );
+
+                return (
+                  <div
+                    key={i}
+                    className="text-blue-50/90"
+                    dangerouslySetInnerHTML={{ __html: formatted }}
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)